index.mjs

```diff
@@ -438,11 +439,16 @@ const jsonConfigurations = [
   },
 ].map((config) => ({ files: ['**/*.json'], ...config }));
 
-const yamlConfigurations = [...yaml.configs['flat/recommended']].map((config) => ({
+const yamlConfigurations = yaml.configs['flat/recommended'].map((config) => ({
   files: ['**/*.yml', '**/*.yaml'],
   ...config,
 }));
 
+const markdownConfigurations = markdown.configs.recommended.map((config) => ({
```

Comment:
supposedly the `processor` config should be able to enable the code linting inside the code blocks, but somehow it doesn't work very well for typescript, and also it doesn't work together with `recommended` configuration either. so for now, we'll only enable [`recommanded` rules](https://github.com/eslint/markdown?tab=readme-ov-file#rules).